pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -137,7 +137,7 @@ func (pl *InterPodAffinity) PreScore(
 	}
 
 	if pl.sharedLister == nil {
-		return framework.NewStatus(framework.Error, fmt.Sprintf("InterPodAffinity PreScore with empty shared lister found"))
+		return framework.NewStatus(framework.Error, "interPodAffinity PreScore with empty shared lister found")
```

Comment:
this is the name of the plugin, so it should start with uppercase. Perhaps you can reorder words:

"empty shared lister in InterPodAffinity PreScore"

Reply:
updated and squashed.